build.gradle

```diff
@@ -55,6 +55,15 @@ allprojects {
         }
     }
 
+    dependencies {
+        constraints {
+            rootConfiguration 'io.dropwizard.metrics:metrics-core', {
+                version { strictly '[3, 4[' }
+                because "Spark still uses 3.X, which can't co-exist with 4.X"
+            }
+        }
+    }
```

Comment:
This snippet encodes your intuition "Not sure we can bump this" in code, so that if anyone ever tries to the build will fail. It's also what we do in WC.

https://github.com/palantir/gradle-consistent-versions#specifying-exact-versions

Reply:
thanks for fixing up @iamdanfox , was just flagging on mobile earlier so this wouldn't merge before getting a chance to fix.